src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -562,6 +570,13 @@ public boolean isOnGround() {
     public void remove() {
         callEvent(PlayerDisconnectEvent.class, new PlayerDisconnectEvent(this));
 
+        this.tabList.removeViewer(this);
+        for (TabList tabList : MinecraftServer.getTabListManager().getTabLists()) {
```

Comment:
There's got to be a better way to do this... Could we keep a map of Player -> TabList in the manager?

Reply:
+1